Taxonomy subtree deletion and clean-up (#2271)
ibexa/documentation-developer: docs/update_and_migration/from_4.4/update_from_4.4.md

    );
    ```

### Clean-up taxonomy database

> 
> If you didn't run it already when [migrating from 4.2 to 4.3](update_from_4.2.md#clean-up-taxonomy-database),
> run the following command for each of your taxonomy,
> to be sure you don't have [Content items orphaned during deletions of subtrees](taxonomy.md#remove-orphaned-content-items) inherited from earlier version's database:
> `php bin/console ibexa:taxonomy:remove-orphaned-content <taxonomy> --force`

Reviewer: ```suggestion

If you didn't run it already when [migrating from 4.2 to 4.3](update_from_4.2.md#clean-up-taxonomy-database), run the following command for each of your taxonomies to ensure that there are no [Content items orphaned during deletion of subtrees](taxonomy.md#remove-orphaned-content-items) inherited from the earlier version's database:

`php bin/console ibexa:taxonomy:remove-orphaned-content <taxonomy> --force`
```
